blog/2023-08-01-tolgee-with-next-13.md

```diff
@@ -0,0 +1,347 @@
+---
+slug: tolgee-with-nextjs-app-router
+title: 'Tolgee with Next.js app router'
+authors: [sgranat]
+description: 'Setting up Tolgee with Next.js new app router and server components.'
+tags: [next.js, server components, react]
+---
+
+With the introduction of Next.js 13 and the app router featuring React server components, there has been a strong push to enable Tolgee to work within this new paradigm.
+
+Until now, Tolgee was designed to function with regular React components. However, we now face the challenge of making in-context translation work with server components. Let's delve into the details.
```

Comment:
There is no intro to the server components, so I don't know how it works.

Reply:
If this is a tutorial, no one is interested into information about us facing some challenge. They just want to know how to get started fast.